tests/unit/ProcessMaker/InboxRules/MatchingTasksTest.php

```diff
@@ -75,24 +76,38 @@ public function testInboxRuleEndDate()
         // not be returned
 
         $user = User::factory()->create();
-        $completedTask = ProcessRequestToken::factory()->create([
+    
+        $task = ProcessRequestToken::factory()->create([
             'user_id' => $user->id,
-            'status' => 'COMPLETED',
+            'status' => 'ACTIVE',
```

Comment:
Why change this to `ACTIVE`? The task we use for InboxRule's process_request_token_id should always be a completed task

Reply:
My mistake, I changed the value to make some tests and I forgot to change it to COMPLETED before the PR.